include/internal/kuzzle_structs.h

```diff
@@ -12,36 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef _KUZZLESDK_H_
-#define _KUZZLESDK_H_
+#ifndef _KUZZLE_STRUCTS_H_
+#define _KUZZLE_STRUCTS_H_
 
 #include <time.h>
 #include <errno.h>
 #include <stdbool.h>
 #include <stdlib.h>
 #include <stdio.h>
 
-enum Mode {AUTO, MANUAL};
+enum KuzzleMode {AUTO, MANUAL};
```

Comment:
I would use a more detailed name as:
```suggestion
enum KuzzleConnectionMode {AUTO, MANUAL};
```